js/coinex.js

```diff
@@ -256,10 +256,6 @@ module.exports = class coinex extends Exchange {
                     'max': undefined,
                 },
             },
-            'precision': {
```

Comment:
why are you removing these lines

Reply:
specifically, for `coinex` there is no need for that lines at all, as the `fetchMarkets` define precisions already. but for other exchanges, where `precision` property is undefefined inside `fetchMarkets`, then I am moving these lines into `fetchMarkets`, because these lines doesnt belong globally to 'exchange', instead to the specific markets (spot, swap, etc...), so they should only be left there when there is not `fetchMarkets` implemented in exchange.